process/block/preprocess/sovereignChainTransactions.go

```diff
@@ -175,20 +177,12 @@ func (sct *sovereignChainTransactions) isTransactionEligibleForExecution(tx *tra
 		return err, false
 	}
 
-	senderAccount, _, errGetAccounts := sct.txProcessor.GetSenderAndReceiverAccounts(tx)
+	senderAccount, accntInfo, errGetAccounts := sct.getSenderAccountInfo(tx)
```

Comment:
what happens in case of relayed v3, when sender does not exist? will this return a nil senderAccount, causing a return false?

it should be a valid case, with a non-existing sender that does a sc call and relayer pays the fee

Reply:
+1

Reply:
This case is covered and works as expected because `sct.txProcessor.GetSenderAndReceiverAccounts` will LoadAccount or CreateAccount if it doesn't exist. 
`senderAccount` is nil only if some error occurs.